app/src/main/java/com/xa/xpensauditor/SetLimitActivity.java

```diff
@@ -22,6 +22,9 @@
 
 import java.util.Calendar;
 
+/**
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First sentence of Javadoc is missing an ending period.